source/common/network/connection_impl.cc

```diff
@@ -336,13 +328,10 @@ void ConnectionImpl::onWriteReady() {
     // It is possible (though unlikely) for the connection to have already been closed during the
```

Comment:
I thought about deleting the comment, but it I figured I would leave it just in case something comes up later, since it took my about 4 hours to figure out what was happening in the first place.